lib/rbac/filterer.rb

```diff
@@ -252,12 +257,36 @@ def search(options = {})
               .includes(include_for_find).includes(exp_includes)
               .order(order)
 
-      scope = include_references(scope, klass, include_for_find, exp_includes, skip_references)
```

Comment:
This is mildly pedantic, but is there a reason this needs to move?  You are doing the:

```ruby
scope.includes_values = []
scope.references_values = []
```

in the `if options[:use_sql_view` block, so that should cancel this out, and avoid needing call this in two different spots now.

Side benefit:  Makes the diff a bit cleaner (minor backporting concern).

Reply:
that sounds good. thanks :+1: